dbinspector/src/main/java/im/dino/dbinspector/activities/DbInspectorActivity.java

```diff
@@ -19,28 +18,28 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.dbinspector_activity);
 
         if (savedInstanceState == null) {
-            FragmentTransaction ft = getFragmentManager().beginTransaction();
-            ft.replace(R.id.dbinspector_container, new DatabaseListFragment()).commit();
+	        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
+	        ft.replace(R.id.dbinspector_container, new DatabaseListFragment()).commit();
         }
     }
 
     @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
-            // Respond to the action bar's Up/Home button
-            case android.R.id.home:
-                onBackPressed();
-                return true;
+    public void onBackPressed() {
```

Comment:
@PrashamTrivedi looks like your code style formatting also changes the method definition order. Please fix this.